lamin_cli/__main__.py

```diff
@@ -328,11 +325,47 @@ def save(path: str, key: str, description: str, stem_uid: str, registry: str):
         sys.exit(1)
 
 
+@main.command()
+@click.argument("filepath", type=str)
```

Comment:
```suggestion
@click.argument("filepath", type=click.Path(exists=True, file_okay=True, readable=True))
```

We could consider something like this. Then the parameter can also become a Path and not a str which makes things clearer and more interoperable across OS etc.

Reply:
That looks cool but is something for a refactor of the entire CLI -- we have a filepath argument in every other command. Let's not start ad hoc changing the convention here for a single command.